lib/storage/data/file/DataFileStore.js

```diff
@@ -151,6 +153,14 @@ class DataFileStore {
                     return cbOnce(null, key);
                 }
                 if (this.noSync) {
+                    /*
+                     * It's is not guaranteed that the Kernel will release page
+                     * caches when this.noSync is true. If you want to ensure
+                     * this behavior, set this.noSync to false.
```

Comment:
You mean set `this.noCache` to true, the comment is confusing otherwise?

Reply:
No, I do mean `this.noSync`. `posix_fadvise` doesn't guarantee to release page caches if the file hasn't been flushed to disk.